lib/steps.js

```diff
@@ -735,6 +737,15 @@ When(/I set the var(?:iable)? "([^"]+)" to "([^"]+)"/, { timeout: -1 }, async (
   );
 });
 
+When(/I set the var(?:iable)? "([^"]+)" to secret "([^"]+)"/, { timeout: -1 }, async ( var_name, set_to_env_name ) => {
```

Comment:
"the" isn't necessary, but we've been trying to make these all very human sentences, so it might be more consistent to allow a "the" in here. I also made it case insensitive. I've been trying to remember to do that more just to give people some wiggle room.

```suggestion
When(/I set the var(?:iable)? "([^"]+)" to ?(?:the)? secret "([^"]+)"/i, { timeout: -1 }, async ( var_name, set_to_env_name ) => {
```

Reply:
Might be worth considering allowing "the GitHub", as in "the GitHub secret", to allow people to give themselves and others some context.

```suggestion
When(/I set the var(?:iable)? "([^"]+)" to ?(?:the)? ?(?:GitHub)? secret "([^"]+)"/i, { timeout: -1 }, async ( var_name, set_to_env_name ) => {
```

Reply:
I'm okay with adding `the`, but `Github` seems a bit to specific, and not that helpful? I guess allowing it is fine.